chains/solana/contracts/programs/ccip-router/src/instructions/v1/fee_quoter.rs

```diff
@@ -249,10 +250,10 @@ pub struct UnpackedDoubleU224 {
 impl From<&TimestampedPackedU224> for UnpackedDoubleU224 {
     fn from(packed: &TimestampedPackedU224) -> Self {
         let mut u128_buffer = [0u8; 16];
-        u128_buffer[2..16].clone_from_slice(&packed.value[14..]);
-        let high = u128::from_be_bytes(u128_buffer);
-        u128_buffer[2..16].clone_from_slice(&packed.value[..14]);
+        u128_buffer[2..16].clone_from_slice(&packed.value[14..]); // @TODO: byte alignment good?
```

Comment:
Great catch that the high/low were getting swapped given that inputs are big-endian, thanks! :+1: 

I think the `TODO` can be removed, the byte alignment seems fine as far as I can tell
```suggestion
        u128_buffer[2..16].clone_from_slice(&packed.value[14..]);
```